wesnoth-wof.po

```diff
@@ -5999,7 +6047,16 @@ msgid ""
 "I would do anything to save my friend.\n"
 "I felt the whole world around me burning for the means to save him."
 msgstr ""
+"Bolestivá vina z té poslední myšlenky...,\n"
+"\n"
+"<i>... nech svého přítele zemřít...,</i>\n"
+"\n"
+"... znovu zapálila mé srdce neznámou vášní.\n"
+"Udělal bych cokoliv, abych svého přítele zachránil.\n"
+"Cítil jsem, jak celý svět kolem mě hoří potenciálem jeho záchrany."
 
+# Ulomený hned pod hlavicí?
+# Tu formulaci o stavu oštěpu jsem nepochopil.
```

Comment:
No, zaprvé díky za kvalitní příspěvky.

Zůstal bych ale u jednoduchého pravidla spear - kopí (na blízko), javelin - oštěp (házení, na dálku). Poznamenal jsem si nutnost opravy v textu drakonidského rváče.

Já bych broken a reach beyond the head přeložil jako kopí se zlomeným ratištěm (tak v půli, hádám, když je to a reach).

V ostatních textech jsem zatím nenašel chybu.

Reply:
Právě že to _spear_ máme i v jiných textových řetězcích přeloženo různě, nejen u rváče. Ale souhlasím.
To zlomené kopí přeložím dle návrhu.